examples/manifests/kustomization.yaml

```diff
@@ -3,16 +3,12 @@ kind: Kustomization
 resources:
 - namespace.yaml
 - service-account.yaml
-- roles.yaml
 - service.yaml
 - consumer.yaml
 replicas:
 - count: 1
   name: cloud-consumer-deployment
 images:
 - name: cloud-event-consumer
-  newName: quay.io/redhat-cne/cloud-event-consumer
-  newTag: latest
-- name: cloud-event-sidecar
-  newName: quay.io/openshift/origin-cloud-event-proxy
-  newTag: latest
+  newName: quay.io/jacding/cloud-event-consumer
+  newTag: remove-v1
```

Comment:
Is it a leftover from private testing?

Reply:
yes, thanks. fixing this in a new PR https://github.com/redhat-cne/cloud-event-proxy/pull/442/files